src/runtime/ltypes/ltypes.py

```diff
@@ -51,3 +51,31 @@ def __call__(self, *args, **kwargs):
 def overload(f):
     overloaded_f = OverloadedFunction(f)
     return overloaded_f
+
+
+
+# C interoperation support
+
+class CTypes:
+    """
+    A wrapper class for interfacing C via ctypes.
+    """
+
+    def __init__(self, f):
+        self.name = f.__name__
+        self.args = f.__code__.co_varnames
+        self.annotations = f.__annotations__
+        lib = "/Users/ondrej/repos/lpython/src/runtime/liblfortran_runtime.dylib"
+        import ctypes
```

Comment:
Can we move this `ctypes` as the class variable so it isn't imported at every object creation?

Reply:
I moved it to the top of the file.